src/latexify/parser_test.py

```diff
@@ -7,6 +7,7 @@
 import pytest
 
 from latexify import exceptions, parser, test_utils
+from latexify.ast_utils import ast_function_def
```

Comment:
We don't allow importing functions/classes directly, import module instead.

Same for every other code.

Reference: https://google.github.io/styleguide/pyguide.html#22-imports

```suggestion
from latexify import ast_utils
```